docs/ssss.md

```diff
@@ -1,3 +1,101 @@
+
+The most common way to implement a DAO is by **using an existing mainnet that
+supports smart contracts**. This involves programming the governance rules of
+the DAO into a smart contract and deploying it on the mainnet. However, this
+approach **has some disadvantages**:
+
+1. Users need to pay **gas fees** to operate the DAO.
+2. It is reliant on the performance, security, reliability, usability, and
+   potential political risks **associated with the chosen mainnet**.
+3. Its capabilities are restricted by the ecosystem of the chosen mainnet and
+   may **not easily interact with other mainnets**.
+
+Simperby solves these problems by providing a standalone blockchain for a DAO.
+
+For more details, refer to [the comparisons (WIP)](./comparison.md)
+
+## Simperby
+
+Simperby is a blockchain engine that builds a standalone blockchain for a DAO.
+
+- Trustless
+- Distributed
+- Fault Tolerant
+- Lightweight
+- Highly Interoperable
+- Native Support for Multichain Treasuries
+
+For details on the core protocol, refer to the [protocol
+overview](./protocol_overview.md).
+
+### A Byzantine Fault Tolerant System
+
+- **Builds an independent layer-1 mainnet** like [Cosmos
+  SDK](https://v1.cosmos.network/sdk) or [Substrate](https://substrate.io/).
+- **One mainnet for one DAO**.
+- **Self-hosted: DAO members maintains the chain**.
+- Has own **BFT consensus** called [Vetomint](./vetomint.md) optimized for
+  sporadic node operations.
+
+### Governance
+
+- A peer-to-peer voting mechanism (governance) is used to make decisions on the
+organization, which is then finalized by the consensus.
+- No tokens, no staking, no mining, no gas fee; instead, **permissioned and
+explicitly nominated nodes**.
+- Has its own peer-to-peer communication channel for the organization.
+
+### Storage
+
+- Provides a distributed file system for the organization.
+- **Managed as a distributed Git repository**.
+  - Blocks and transactions are stored as Git commits.
+  - Governance and consensus proposals are stored as Git branches.
+  - The canonical history of the chain is presented as a designated Git branch.
+
+### Interoperability
+
+- Native support for trustless message delivery that establishes **multichain
+  interoperability** directly.
+- Based on a light client, which verifies the finality of the chain without
+  requiring the full state or full block, sharing the same principle as [Cosmos
+  IBC](https://ibc.cosmos.network/).
+- Easy implementation over various existing mainnets, thanks to its simple and
+  lightweight architecture.
+- Effective control of treasuries, tokens, and other dapps by the organization
+  without requiring trusted third parties.
+- **Treasury contracts** for the supported mainnets are available out of the
+  box.
+
+### Sporadic & Lightweight Node Operation
+
+- Simperby's novel consensus protocol allows nodes to be operated sporadically;
+  **nodes are not required to operate 24/7**. Participants may turn on their
```

Comment:
"nodes are operated" vs "nodes operate"?
I'm not sure which is correct but we are mixing up the two in this sentence.

Reply:
fixed.